src/System.Reflection.Emit/tests/AssemblyBuilder/AssemblyBuilderGetManifestResourceStream2.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+using System.Reflection;
+using System.Reflection.Emit;
+using System.IO;
+using Xunit;
+
+namespace System.Reflection.Emit.Tests
+{
+    public class AssemblyBuilderGetManifestResourceStream2
+    {
+        private const AssemblyBuilderAccess DefaultBuilderAccess = AssemblyBuilderAccess.Run;
```

Comment:
Seems like an unnecessary constant, as it's only used once and it's already a named enum value, but not a big deal.